src/Components/GEOSldas_GridComp/GEOSlandassim_GridComp/clsm_ensupd_upd_routines.F90

```diff
@@ -3500,7 +3532,20 @@ subroutine cat_enkf_increments(                               &
     real, dimension(     N_catd)          :: tp1_ensavg
 
     type(obs_param_type)                  :: this_obs_param
-        
+   
+    !jpark50
+    real, dimension(N_catd, N_ens)        :: wesn_sum_old
+    real, dimension(N_catd, N_ens)        :: areasc_old 
+    real, parameter:: cpw   = 2065.22 !ice specific heat capacity @ 0C [J/kg/K]
+    real, parameter:: model_threshold = 0.40
+    real, parameter:: obs_threshold = 0.25
+    real, parameter:: wesn_added = 12.0 ! mm of SWE
+    real, parameter:: eps = 1.e-4
+    real, parameter:: small = 1.e-20
+    real, parameter:: rhofs = 150. ! kg/m^3 density of fresh snow
+    real, parameter:: rhoma = 500. ! kg/m^3 maximum snow density
+    real, parameter:: wemin = 13. ! kg/m^3
```

Comment:
These parameters are used in the Stieglitz snow model and should be coming from there (rather than defined again here).  This should work with "use" statements as follows:

USE SurfParams,                 ONLY: WEMIN
USE StieglitzSnow,              ONLY: RHOMA, CPW
USE CATCH_CONSTANTS,   ONLY: RHOFS  => CATCH_SNWALB_RHOFS